SubZero/src/model/FPGAModel.h

```diff
@@ -9,12 +9,15 @@
 #define FPGAMODEL_H_
 
 #include "Model.h"
+#include "FPGAState.h"
 
 /**
  * This is a concrete child of the Model class. It implements functions
  * specific to communicating with the FGPA.
  */
 class FPGAModel: public Model {
+private:
+	FPGAState* fpgaState;
```

Comment:
@PyroIgnus Right now there is no "parent state" class. I will make parent Model class to include "observable" class then make them into specific state in children Model classes. Is that OK?